app/assets/javascripts/stores/student_store.es6.jsx

```diff
@@ -13,19 +14,30 @@
         student_conference: {},
       };
       this.bindListeners({
+        handleStoreComment: StudentActions.STORE_COMMENT,
+        handleStoreComments: StudentActions.STORE_COMMENTS,
         handleStoreOverlay: StudentActions.STORE_OVERLAY,
         handleStoreStudent: StudentActions.STORE_STUDENT,
         handleToggleSidebar: StudentActions.TOGGLE_SIDEBAR,
       });
     }
 
+    handleStoreComment(response) {
+      this.comments.push(response.comment);
```

Comment:
This is a cute fix of the problem we had before @maxwolffe, it turns out alt figures out there's a change!